dnf5/context.cpp

```diff
@@ -257,6 +258,111 @@ void Context::load_repos(bool load_system, libdnf::repo::Repo::LoadFlags flags)
 }
 
 
+namespace {
+
+class FileDownloadCB : public libdnf::repo::FileDownloadCallbacks {
+public:
+    FileDownloadCB(libdnf::cli::progressbar::MultiProgressBar & mp_bar, const std::string & what)
+        : multi_progress_bar(&mp_bar),
+          what(what) {
+        progress_bar = std::make_unique<libdnf::cli::progressbar::DownloadProgressBar>(-1, what);
+        multi_progress_bar->add_bar(progress_bar.get());
+    }
+
+    int end(TransferStatus status, const char * msg) override {
+        switch (status) {
+            case TransferStatus::SUCCESSFUL:
+                //std::cout << "[DONE] " << what << std::endl;
```

Comment:
Did you leave these debug lines in intentionally?

Reply:
I think I can remove them now.